les/api_backend.go

```diff
@@ -290,5 +289,5 @@ func (b *LesApiBackend) GatewayFeeRecipient() common.Address {
 
 func (b *LesApiBackend) GatewayFee() *big.Int {
 	// TODO(nategraf): Create a method to fetch the gateway fee values of peers along with the coinbase.
-	return eth.DefaultConfig.GatewayFee
+	return big.NewInt(23)
```

Comment:
Please clean up this change. I don't recall immediatly what should be here, but it should not be a constant.